README.md

```diff
@@ -1,24 +1,247 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> The following steps were used in creating this project:
+
+1 Setup the project by creating a new directory from the generated repository.
+
+2 Set up the repository on GitHub and use Gitflow.
+
+3 Initialized project with rubocop linters
+
+> To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+1 Ruby installed (Using Homebrew on macOS machine)
+
+```sh
+brew install ruby
+```
+
+2 Rails Installed
+
+```sh
+gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/AlierPM/blog.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd blog
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+   cd blog
+```
+```sh
+   code .
+```
+
+### Run project
+
+To run from the local server, run the following command:
+
+```sh
+  rails server
+```
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rpec spec
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Authors
+
+👤 **Alier Philip Maguet**
+
+- GitHub: [@AlierPM](https://github.com/AlierPM)
+- Twitter: [@AlierPM](https://twitter.com/AlierPM)
+- LinkedIn: [@AlierPM](https://www.linkedin.com/in/alierphilipmaguet/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features that will be added to the project.
+
+- [x] **Add user login UI**
+- [x] **Add sign up page**
+- [x] **Add sharing of posts option**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Your contructive reviews help us to improve progressively
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+>[Microverse](https://www.microverse.org/?grsf=0gzf37)
+
+I would like to thank the Microverse for offering this topic notch course that has equiped me to be a full stack Ruby on Rails Developer
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ <a name="faq"></a>
+
+> You want to know! Relax! We got you!
+
+- **How Long did it take to build this project?**
+
+  - [x] 5 days
+
+- **How many git conflicts did you resolve?**
+
+  - [x] More than five (5) conflicts resolved
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ Optional ] Kindly I recommend you to Fix the MIT License , the link is broken as shown below with the following Screenshot.  
![image](https://github.com/AlierPM/BlogPress/assets/93598189/4ce0436f-992d-4536-9659-9a032203c822)

Reply:
Noted, thanks @Abenezer-Tilahun